lib/graphql/language/lexer.rb

```diff
@@ -653,15 +671,24 @@ def self.emit(token_name, ts, te, meta)
 
       def self.emit_string(ts, te, meta)
         value = meta[:data][ts...te].pack("c*").force_encoding("UTF-8")
-        replace_escaped_characters_in_place(value)
-
-        meta[:tokens] << GraphQL::Language::Token.new(
-          name: :STRING,
-          value: value,
-          line: meta[:line],
-          col: meta[:col],
-        )
-        meta[:col] += te - ts
+        if value =~ /\\u|\\./ && value !~ ESCAPES
+          meta[:tokens] << GraphQL::Language::Token.new(
+            name: :BAD_UNICODE_ESCAPE,
+            value: value,
+            line: meta[:line],
+            col: meta[:col],
+          )
+        else
```

Comment:
do we need to increment the `meta[:col]` counter in this case? I see that the `else` case has `meta[:col] += te - ts` which keeps the column number on track, do we need it here too?